components/shared_server_code/pyproject.toml

```diff
@@ -1,24 +1,6 @@
-[build-system]
-requires = [
-    "setuptools==67.6.1",
-    "wheel==0.40.0"
-]
-build-backend = "setuptools.build_meta"
-
 [project]
 name = "shared_server_code"
 version = "4.8.0"
-authors = [
-  { name="Quality-time team", email="quality-time@ictu.nl" },
-]
-readme = "README.md"
-requires-python = ">=3.10"
```

Comment:
Why no longer include this?

Reply:
Because we don't publish these packages.